### PR TITLE
Uml 1194 older lpa fixes

### PR DESCRIPTION
**Pull request by cooperaj**
## Purpose

Implement the letter request API into the mock service and fix the json interpretation of body content so it passes it along correctly.

## Checklist

* [x] I have performed a self-review of my own code
* [ ] ~I have added relevant logging with appropriate levels to my code~
* [ ] ~I have updated documentation where relevant~
* [x] I have added tests to prove my work
* [x] I have run the integration tests (results below)

### CODE REVIEW

File: lambda_functions/v1/requirements/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #Update this date to trigger update of layers: 20200807
 Flask==1.1.2
 Werkzeug==1.0.1
-pyjwt==2.0.0
+pyjwt==1.7.1
```
Comment (jamesrwarren) [inline lambda_functions/v1/requirements/requirements.txt:4]: I believe you can take this out entirely now that you have dependency in opg-data. Tried it without locally yesterday and it worked without.

Comment (cooperaj) [inline lambda_functions/v1/requirements/requirements.txt:4]:
```suggestion
```

File: lambda_functions/v1/requirements/dev-requirements.txt [hunk 2]
```diff
@@ -9,12 +9,12 @@ pytest-cov==2.10.1
 requests-aws4auth==1.0.1
 pytest-html==3.1.1
 moto
-pyjwt==2.0.0
+pyjwt==1.7.1
```
Comment (cooperaj) [inline lambda_functions/v1/requirements/dev-requirements.txt:12]:
```suggestion
```

**Review by jamesrwarren — APPROVED** [review]

**Comment by cooperaj** [comment]
Screenshot 2021-01-13 at 13 30 51

**Review by jamesrwarren — APPROVED** [review]

**Pull request merged**